src/test/e2e-emulated/send.spec.ts

```diff
@@ -61,4 +61,66 @@ describe("send", () => {
             exitCode: 130,
         });
     });
+
+    it("should send with intermediate reservations", async () => {
+        // emit, nativeReserve, send
+        let balanceBefore = await contract.getBalance();
+        await expectMessageFromTo(
+            treasure,
+            contract,
+            textMsg("ReserveAtMost_1"),
+        );
+        let balanceAfter = await contract.getBalance();
+        // The difference is exactly 0.05 that were reserved on top of the balance
+        expect(abs(balanceBefore - balanceAfter) <= 50000000n).toBe(true);
```

Comment:
Let's remove `abs`, it seems in this case it should be clear if the balance "after" is smaller or greater than "before"